### PR TITLE
Feature/#87  wtp change

### LINKED ISSUE #87: Create DiscreteChoice.wtp_change()
Using the output from .diversions(), calculate the WTP change between a list of combined choices.

### PR DESCRIPTION
**Pull request by suhail-thahir-sat3ba**
closes #87

### CODE REVIEW

File: pymanda/choices.py [hunk 1]
```diff
@@ -750,4 +750,51 @@ def diversion(self, cd, choice_probs, div_choices, div_choices_var=None):
...
+            raise TypeError ('''trans_list expected type list. got {}'''.format(type(trans_list)))
+        
+        if len(trans_list) < 2:
+            raise ValueError ('''trans_list needs atleast 2 choices''')
+        
+        for tran in trans_list:
+            if tran not in choice_probs.columns:
+                raise KeyError ('''{} is not a choice in choice_probs'''.format(tran))
+        
+        if self.solver == 'semiparametric':
```
Comment (bperry12) [inline pymanda/choices.py:786]: why do we need this if statement? this should work no matter what solver we use, right?

File: pymanda/choices.py [hunk 2]
```diff
@@ -750,4 +750,51 @@ def diversion(self, cd, choice_probs, div_choices, div_choices_var=None):
...
+                warnings.warn('''A diversion probability for a bin equals 1 which will result in infinite WTP.''' , RuntimeWarning)
+            
+            with warnings.catch_warnings(record = True): # prevents redundant warning for np.log(0)
+                wtp_df = -1 * np.log(1- wtp_df) # -1 * ln(1-prob)
+            
+            wtp_df = wtp_df.sum().to_frame().transpose()
+            
+            wtp_df['wtp_change'] = (wtp_df['combined'] - wtp_df[trans_list].sum(axis = 1)) /  wtp_df[trans_list].sum(axis = 1)
+        
+        return wtp_df
```
Comment (bperry12) [inline pymanda/choices.py:800]: we usually create a table with WTP for each system separately, then combined, and then the change. i think we need to output more than what you have here in order to make this, yes?

**Review by bperry12 — CHANGES_REQUESTED** [review]

File: pymanda/choices.py (hunk 2 above)
Comment (suhail-thahir-sat3ba) [reply pymanda/choices.py:800]: That is what is currently being outputted. An example of the output is in test_pymanda, lines 523-526.

File: pymanda/choices.py (hunk 1 above)
Comment (suhail-thahir-sat3ba) [reply pymanda/choices.py:786]: Correct, will remove

**Review by bperry12 — APPROVED** [review]

**Pull request merged**